[hotfix] [docs] Fix typo in checkpoint monitoring doc (#13158)
apache/flink: docs/monitoring/checkpoint_monitoring.zh.md

- **Trigger Time**: The time when the checkpoint was triggered at the JobManager.
- **Latest Acknowledgement**: The time when the latest acknowledged for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
- **End to End Duration**: The duration from the trigger timestamp until the latest acknowledgement (or n/a if no acknowledgement received yet). This end to end duration for a complete checkpoint is determined by the last subtask that acknowledges the checkpoint. This time is usually larger than single subtasks need to actually checkpoint the state.

> - **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. This value would be the delta delta checkpointed data if incremental checkpoint is on.

Reviewer: ```suggestion
- **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. If incremental checkpointing is enabled this value is the checkpointed data size delta.
```

Author: @zentol  updated per suggested
